docs/how-to/backup-and-restore.md

```diff
@@ -0,0 +1,129 @@
+# How to back up and restore Synapse
+
+This document shows how to back up and restore Synapse.
+
+A backup strategy is an integral part of disaster and recovery, and should be
+planned accordingly. The main goal of a backup is the possibility of being
+restored and the backup and restore process should be tested as part of
```

Comment:
This phrasing sounds a bit weird. Perhaps it can be removed?

Reply:
This, specifically `The main goal of a backup is the possibility of being
restored and the backup and restore process should be tested`

Reply:
Removed everything. Maybe for another place...